tests/estimators/speech_recognition/test_pytorch_deep_speech.py

```diff
@@ -0,0 +1,396 @@
+# MIT License
```

Comment:
Could you please move these tests to the new pytest pattern?

Reply:
Yes, done.